src/assets/styles/grid/_variables.scss

```diff
@@ -1,29 +1,18 @@
-$breakPoints: (
-        xs: 0,
-        sm: 576px,
-        md: 768px,
-        lg: 992px,
-        xl: 1200px,
-) !default;
-
-$gridColumns: 12 !default;
-$gridRowGap: 16px !default;
-$gridColumnGap: 16px !default;
-$columnPrefix: '.col' !default;
-$alignPrefix: '.align' !default;
-
-$alignments: start, end, center, stretch;
+$grid-columns: 12 !default;
```

Comment:
looks like old grid were accidentally merged on the top of the new one

Reply:
unused 
![image](https://github.com/user-attachments/assets/b7a27f47-f41e-49df-8c94-af59fb224125)